linux_os/guide/system/accounts/accounts-banners/banner_etc_issue_net/ansible/shared.yml

```diff
@@ -0,0 +1,11 @@
+# platform = multi_platform_fedora,multi_platform_ol,multi_platform_rhel,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
```

Comment:
```suggestion
# platform = multi_platform_all
```

Reply:
Good. I will include in the next update.